lint/src/main/kotlin/net/grandcentrix/thirtyinch/MissingViewInCompositeDetector.kt

```diff
@@ -0,0 +1,94 @@
+package net.grandcentrix.thirtyinch
+
+import com.android.annotations.VisibleForTesting
+import com.android.tools.lint.detector.api.Issue
+import com.android.tools.lint.detector.api.JavaContext
+import com.intellij.psi.PsiClass
+import com.intellij.psi.PsiClassType
+import com.intellij.psi.PsiJavaCodeReferenceElement
+import com.intellij.psi.PsiType
+import org.jetbrains.uast.*
```

Comment:
AS' default Kotlin code style encourages wildcard imports when at least 5 members of a class are being used. I have adjusted the IDE's settings and rearranged the import statements.